.github/workflows/build.yml

```diff
@@ -19,3 +19,23 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  gofmt:
+    name: "Run gofmt check"
+    runs-on: ubuntu-latest
+    steps:
+      -
```

Comment:
@cristiGuranIonos not sure if this affects the functionality but shouldn't be `- name`, I mean the dash shouldn't be before `name` keyword?

Reply:
it can be on the previous line also. moved all `-` to same line